### ISSUE TITLE
Vampirism causes massives amount of FPS drop during aoe

### ISSUE THREAD

**Issue by lyssebabz** [slowtorta/turtlewow-bug-tracker#4689]
**Please, provide a clear description what the bug is:**
When i use 5% vampirism (i reckon any % of vampirism would replicate) and pulls 40 mobs, it makes my fps go to basically 0 when i do any form of aoe. 

**Steps to reproduce the behavior with as much detail as possible:**
Use vampirism
https://streamable.com/c6vnpz Video with 5% vampirism with huge fps drop (2 rings and bracer enchant)
https://streamable.com/b7xa7g using 4% vampirism with huge fps drop (2 rings)
https://streamable.com/wj7cha using 3% a mix of ring and bracer enchant, huuge fps drop (ring and bracer)
https://streamable.com/l2a1nf Using 2% vampirism still stuttering, but playable (ring only)
https://streamable.com/6bn31e using 1% vampirism enchant, bit stuttering but still playable (bracer only)
https://streamable.com/k52bom using 0% vampirism, i can freely move around after using my concecration (nothing)

in conclusion: more vampirism = more lag. Using 1 item of either 1 or 2 % seems to work fine, bit stuttering but it works as i would assume it would. mixing and matching multiple items seem to cause massive amount of fps drop

https://www.youtube.com/watch?v=J1ganLuQgaQ&t=162s&ab_channel=Lyssebabz  Pre 1.17, still a bit fps drop but far better than the 2 mentioned above

**Expected behavior. Describe how it should work:**
It should not tank you fps during aoe spells like concecrartion, using concecration on 40 mobs pretty much makes it so you cannot play for 8  seconds. It does still take input as seen in the video above. I've tried using the FOV client and normal client, both with same result

**Comment by ratkosrb** [comment]
This will be fixed in the next update. The healing from Vampirism will be batched in 1 second intervals.

**Comment by lyssebabz** [comment]
> This will be fixed in the next update. The healing from Vampirism will be batched in 1 second intervals.

Does that mean during reset? or next patch update and if so, can you give an ETA when that rolls out
